pkg/process/checks/process.go

```diff
@@ -82,11 +86,29 @@ func (p *ProcessCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.Mess
 		return nil, errEmptyCPUTime
 	}
 
+	var sysProbeUtil *net.RemoteSysProbeUtil
+	// if the Process module is disabled, we allow Probe to collect
+	// fields that require elevated permission to collect with best effort
+	if !cfg.CheckIsEnabled(config.ProcessModuleCheckName) {
+		procutil.WithPermission(true)(p.probe)
+	} else {
+		procutil.WithPermission(false)(p.probe)
+		if pu, err := net.GetRemoteSystemProbeUtil(); err == nil {
```

Comment:
Is there any overhead in calling `GetRemoteSystemProbeUtil` every check run instead of doing it once in the `Init` function?

Reply:
I think the overhead is pretty minimal. The availability is checked in the background and every check that uses system-probe data has this in `Run()` function.

Reply:
Thanks for clarifying it!